docs/iceberg-rest-service-blog.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Gravitino Iceberg REST catalog service"
+slug: /iceberg-rest-service-blog
+keywords:
+  - Iceberg REST catalog
+license: "Copyright 2023 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+## Background
+[Apache Iceberg](https://iceberg.apache.org/) is a popular high-performance format for huge analytic tables. Iceberg catalogs is responsible for creating, dropping, alerting Iceberg tables and namespaces. The catalog stores a reference to the current table metadata, and provides the grant of atomic update to the current table metadata. Iceberg catalog is completely pluggable and provides several implements:
+1. Hive catalog, tracks namespaces and tables using a Hive metastore.
+2. JDBC catalog, tracks namespaces and tables in a simple JDBC database like MySQL or PostGreSQL.
+3. HADOOP catalog, tracks namespaces and tables in HDFS.
+ 
+All above catalogs implement all catalog logics in client side, challenges persist:
+1. **Client compatibility**: Client is responsible for compatibility.
+2. **Language and Ecosystem Compatibility**: The current Java implementation poses challenges for maintenance in other languages and native engines.
+3. **Consistency**: Consistency across implementations.
+4. **Performance Limitations**: Opportunities for performance enhancements remain underexplored.
+
+In response, the Iceberg 0.14.0 release ushered in the REST catalog, outlined in the [Iceberg REST API specification](https://github.com/apache/iceberg/blob/main/open-api/rest-catalog-open-api.yaml). This introduces a paradigm shift, placing the bulk of logic on the server side, making the client lightweight.
```

Comment:
...the Iceberg 0.14.0 release supports a REST catalog, as outlined....
This places the bulk of the logic on the server side, making for lightweight clients.